smartcar/exceptions.py

```diff
@@ -18,6 +18,32 @@ def __str__(self):
         return self.message
 
 
+class SmartcarExceptionV2(Exception):
+    """ Exceptions throw by v2.0 endpoints """
+
+    def __init__(self, response):
+        if type(response) is requests.models.Response:
+            json = response.json()
+            if "type" in json:
+                self.type = json["type"]
```

Comment:
Im making the assumption that if `type` exists then so do all the other fields.  If there's no `resolution` or `detail` then are we sending back empty values or just not including the properties on the errors?

Reply:
For now the team decided to send back `null` instead of not including the attribute in the response